routingtable/client_interface.go

```diff
@@ -37,4 +38,5 @@ type AdjRIBOut interface {
 	AddPathInitialDump(pfx *net.Prefix, path *route.Path) error
 	ReplacePath(*net.Prefix, *route.Path, *route.Path)
 	RefreshRoute(*net.Prefix, []*route.Path)
+	Destroy()
```

Comment:
What semantics are bound to this request? What should an implementor of this interface guarantee after the `Destroy` method is called?

Reply:
Added comment